GitHub.Collectors/Cache/EventsBookkeeper.cs

```diff
@@ -36,7 +37,7 @@ public async Task InitializeAsync()
             }
 
             this.table = await AzureHelpers.GetStorageTableAsync("eventstats").ConfigureAwait(false);
-            this.queue = await AzureHelpers.GetStorageQueueAsync("eventstats").ConfigureAwait(false);
+            this.queue = await AzureHelpers.GetStorageQueueUsingMsiAsync("eventstats", storageAccountNameEnvironmentVariable).ConfigureAwait(false);
```

Comment:
It really depends on the internal logic of each processor / collector, but I agree that it might be nice to go over at some point to see what abstraction / clean-up can be done.